### ISSUE TITLE
Clarification on Dataset Usage in the Code

### ISSUE THREAD

**Issue by sanokows** [VsonicV/es-fine-tuning-paper#45]
Hi,
I noticed the following lines in the code (es_fine-tuning_countdown_accl.py) and wanted to clarify their purpose:

# Load data
data_path = "countdown/data/countdown.json"
with open(data_path, "r") as f:
    task_datas = json.load(f)
task_datas = task_datas[:200]

Does this mean the model is only trained on the first 200 samples from the dataset, rather than the full training set? Is this approach specific to this codebase, or is it also reflected in the paper?
Thanks for your help!

**Comment by dibbla** [comment]
@VsonicV

**Comment by VsonicV** [comment]
@sanokows Hi, yes, we only use the first 200 samples for training, and the remaining 2000 for testing. This detail has been clarified in A.2. Experimental Setup in the appendix in the original paper.

**Comment by sanokows** [comment]
Thank you for the clarification. 
I am a bit suprised that relatively to the test samples only a few training samples are used.
As, I am not a domain expert in LLM RLHF I am wondering if this is a common procedure to do the RLHF finetuning?

**Comment by VsonicV** [comment]
@sanokows The countdown task itself is not exactly RLHF, which means fine-tuning model based on human feedback. The countdown task is a case in RLVR (VR stands for verified reward). However, applying ES to RLHF scenario should not make too much difference: you simply need to evaluate each response with a reward model, then do the same ES fine-tuning.

**Comment by sanokows** [comment]
True, thank you for the clarifications!
